db/mob_chat_db.yml

```diff
@@ -0,0 +1,158 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2021 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Monster Chat Database
+###########################################################################
+#
+# Settings
+#
+###########################################################################
+# - Id                      Index of the message.
+#   Color                   Hexadecimal color.
+#   Dialog                  Text displayed by the monster.
+###########################################################################
+
+Header:
+  Type: MOB_CHAT_DB
+  Version: 1
+
+Body:
+  - Id: 1
+    Color: 0xFF0000
```

Comment:
Sorry if I might be blind and one value is different, but maybe make it default 0xFF0000?